src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -281,6 +281,19 @@ async function validate_s3_policy(policy, bucket_name, get_account_handler) {
     }
 }
 
+/**
+ * CRUD action on bucket policy only allow to bucket owner.
+ * @param {string} bucket_owner_id - bucket owner id
+ * @param {string} account_id - account id
```

Comment:
I think you can remove the details, the type and the variable name to give us the needed information.
```suggestion
 * @param {string} bucket_owner_id
 * @param {string} account_id
```